python/tests/test_nn.py

```diff
@@ -19,6 +19,28 @@ def test_linear(self):
         outputs = layer(inputs)
         self.assertEqual(tuple(outputs.shape), (10, 8))
 
+    def test_int8_quantize(self):
+        inputs = mx.full((2, 2), 2.0, dtype=mx.float32)
+        new_inputs, scale = nn.absmax_quantize(inputs)
+        res = new_inputs * scale
+        assert np.allclose(res, inputs)
+
+    def test_linear_quantize(self):
```

Comment:
test_linear_quantize and subsequent tests only operate on a small input size of 2x2. Testing on larger realistic sizes would provide more confidence.

Reply:
could expand it to cover more cases, but it would be duplicating the int8_quantize tests as its doing the correctness of bringing back the original values.